test/addressfield-test.js

```diff
@@ -336,6 +336,37 @@
     expect(17);
   });
 
+  test('select a default country', function() {
+    var $cloneAddress = this.address.clone(),
+        $emptyCountry = $cloneAddress.find('.country'),
+        mockJsonData = {
+          "label": "Country",
+          "options": [
+            {
+              "label": "United States",
+              "iso": "US"
+            },
+            {
+              "label": "Canada",
+              "iso": "CA"
+            },
+            {
+              "label": "Japan",
+              "iso": "JA"
+            }
+          ]
+        };
+
+    $emptyCountry.attr("data-country-selected", "US");
```

Comment:
Something is not being called right because I cannot get this test to pass when I switch it to CA. I cannot find, where in the test case that the plugin is being instantiated. Could you help?